backend/src/appointment/templates/email/includes/base.jinja2

```diff
@@ -8,7 +8,7 @@
 {% set colour_tbpro_apmt_primary = '#008080' %}
 {% set colour_tbpro_apmt_primary_hover = '#066769' %}
 {% set colour_tbpro_apmt_secondary = '#81D4B5' %}
-<html lang="{{ l10n('locale') }}" style="
+<html lang="{{ l10n('locale', lang=lang if lang else None) }}" style="
```

Comment:
Since we need to translate the base (and footer) templates too, but not all emails require a custom locale, I did implement it like this. If you have a better idea, please tell!